package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "Extension Activity Monitor provides more transparency which could help to increase the reliability of abuse reporting and accountability on developers. It is an additional useful tool to aid investigating bugs in the extensions or in the Firefox WebExtensions internals.",
   "scripts": {
+    "start:firefox": "web-ext run -s src --firefox-binary /path/to/firefox-nightly/firefox --pref=\"extensions.experiments.enabled=true\"",
```

Comment:
You don't have to manually specify the binary location. If you use `-f nightly` (`-f` is short for `--firefox-binary`), then `web-ext` (or rather its dependency at https://github.com/mozilla-jetpack/node-fx-runner ) will locate the binary. You only need to set a path if it is not in a standard location. But that user-specific location should not be hard-coded in the `package.json` file, because it will likely not work for anyone else.